app/components/classroom/actions.js

```diff
@@ -0,0 +1,151 @@
+import _ from 'lodash';
+
+import {actions} from './constants';
+import {addClassroomData, getClassroomData, deleteClassroom, updateRoomData} from '../../collections';
+import {inputValidation} from '../helpers/formValidation';
+
+export function changeClassroomTab(event) {
+    return dispatch => {
+        const tabButtons = {classTab: '', examTab: ''};
+
+        const tabUpdate = _.set(
```

Comment:
@hcuffy this works, but keep in mind that `set` mutates the object: https://lodash.com/docs/4.17.15#set